test/data/ansible/check_ostree_nort.yaml

```diff
@@ -324,7 +324,29 @@
 
     # case: check installed greenboot packages
     # https://github.com/osbuild/osbuild-composer/blob/master/internal/distro/rhel8/distro.go#L634
-    - name: greenboot and it's related packages should be installed
+    - name: greenboot and it's related packages should be installed (RHEL 8.6+ and CS9)
+      block:
+        - name: greenboot and it's related packages should be installed
+          shell: rpm -qa | grep greenboot
+          register: result_greenboot_packages
+
+        - assert:
+            that:
+              - "'greenboot-0' in result_greenboot_packages.stdout"
+              - "'greenboot-default-healthchecks' in result_greenboot_packages.stdout"
```

Comment:
isn't the new package `greenboot-default-health-checks` with an extra dash?

Reply:
same above in the other file

Reply:
Yes, thank you.  Silly mistake.

Reply:
Also why does the base greenboot package listed as 'greenboot-0'? If we bump to 1.0 it breaks because the match would need to be 'greenboot-1'

Reply:
It's just doing substring matching.  When the version changes to `1.*`, we'll have to change the test, yes.  The reason is that `greenboot-` would match other packages (the `greenboot-default-healthchecks` package for example) and we want to be sure that we're matching the core greenboot package.

Reply:
IMO it is acceptable to have the version of the package hard-coded in the assertion if that's the easiest way to implement it. The assertion will fail when the package version changes and will require a patch. From QE's POV this is fine if it allows us to have the assertions written in a very straight forward way. 

OTOH if there is another way to implement this assertion and match exactly for package name (I'm not familiar with this piece of code so IDK) let's go for it.

In any case I don't think we should block or delay this PR only because of how the assertions are written.